jdbc/src/main/java/org/apache/iotdb/jdbc/ResultSetMetaDataAdapter.java

```diff
@@ -0,0 +1,103 @@
+package org.apache.iotdb.jdbc;
```

Comment:
apache license?

Reply:
These kinds of modifications will be revoked.